app/src/main/java/org/feature/fox/coffee_counter/data/repository/ItemRepository.kt

```diff
@@ -15,6 +14,10 @@ class ItemRepository @Inject constructor(
     private val itemDao: ItemDao,
     private val apiService: ApiService,
 ) : ItemRepositoryInt {
+    private object ErrorConstants {
+        const val REACH_SERVER_ERROR = "Could not reach the server."
+        const val UNKNOWN_ERROR = "Unknown error occurred."
+    }
```

Comment:
Think those `ErrorConstants` could be moved into Gradle File like other constants

Reply:
Shoudl be resolved in [91ac67f](https://github.com/Morphclue/code-camp-2/pull/69/commits/91ac67f65c01cc4a5a4e52afbae08b130e67f8e8).